spec/controllers/quotes_controller_spec.rb

```diff
@@ -125,6 +125,7 @@
       @company_one = create(:company)
       @company_two = create(:company)
       @company_three = create(:company)
+      @company_four = create(:company, :without_credits)
```

Comment:
we don't need to tuck things that happen only once into before each. just create the company within the scope of the test